### PR TITLE
Adding UI schema to handle form dialogs

### PR DESCRIPTION
**Pull request by GeoffCoxMSFT**
This assists form dialogs in Composer - https://github.com/microsoft/BotFramework-Composer/issues/4360

## Description
This adds missing UI schema for Microsoft.Ask, Microsoft.OnAssignEntity, and Microsoft.OnEndOfActions necessary to have them render property in composers flow designer and property pane.

## Specific Changes
  - Add UI schemas

## Testing
N/A

### CODE REVIEW

File: libraries/Microsoft.Bot.Builder.Dialogs.Adaptive/Schemas/TriggerConditions/Microsoft.OnAssignEntity.uischema [hunk 1]
```diff
@@ -0,0 +1,14 @@
+{
+    "$schema": "https://schemas.botframework.com/schemas/ui/v1.0/ui.schema",
+    "form": {
+        "order": [
+            "condition",
+            "*"
+        ],
+        "hidden": [
+            "actions"
+        ],
```
Comment (chrimc62) [inline libraries/Microsoft.Bot.Builder.Dialogs.Adaptive/Schemas/TriggerConditions/Microsoft.OnAssignEntity.uischema:10]: Does this mean hidden on the right property pane and only in the design surface?

**Review by chrimc62 — APPROVED** [review]
:shipit:

File: libraries/Microsoft.Bot.Builder.Dialogs.Adaptive/Schemas/TriggerConditions/Microsoft.OnAssignEntity.uischema (hunk 1 above)
Comment (GeoffCoxMSFT) [reply libraries/Microsoft.Bot.Builder.Dialogs.Adaptive/Schemas/TriggerConditions/Microsoft.OnAssignEntity.uischema:10]: Correct

**Review by tomlm — APPROVED** [review]
:shipit:

**Pull request merged**